ci/configure-vm.sh

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env bash
+
+set -eu
+
+function checks() {
+    if [[ $(id -u) -ne 0 ]]; then
+        echo "[-] This script needs to be run as root!"
+        exit 1
+    fi
+
+    if [[ $# -eq 0 ]]; then
+        echo "[-] User name of the runner user account required as first argument!"
+        exit 1
+    else
+        username=$1
+    fi
+    workspace=/home/$username/actions-runner
+
+    # shellcheck disable=SC1091
+    source /usr/lib/os-release
+    case "$ID" in
+        fedora) ;;
+        *)
+            echo "[-] Support for $NAME has not been added to this script!"
+            exit 1
+            ;;
+    esac
+
+    if [[ ! -f $workspace/svc.sh ]]; then
+        echo "[-] Runner must be configured already!"
+        echo '[-] Follow all "Download" steps and the first "Configure" step on the "Create self-hosted runner" page.'
```

Comment:
Can you print a link to the page?

Reply:
Done: 1e4b49f9ae79af55b26b8a7c1c21ed335dde4df8